modules/apps/blogs/blogs-service/src/main/java/com/liferay/blogs/internal/upgrade/BlogsServiceUpgrade.java

```diff
@@ -71,8 +71,7 @@ public void register(Registry registry) {
 			new UpgradeDiscussionSubscriptionClassName(
 				_assetEntryLocalService, _classNameLocalService,
 				_subscriptionLocalService, BlogsEntry.class.getName(),
-				UpgradeDiscussionSubscriptionClassName.DeletionMode.CUSTOM,
-				_retainGroupSubscriptions()));
+				UpgradeDiscussionSubscriptionClassName.DeletionMode.UPDATE));
```

Comment:
hi @robertoDiaz if the subscriptions get updated (the classNameId, that is), they won't be visible to the User any more as the "Subscribe" buttons state depends on whether the Blog subscription is present. So, the custom deletion method is needed here as well.